app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,60 @@
+<section class="recipe-main">
+  <h1 class="detail-title"><%= @recipe.name %></h1>
+
+
+
+  <div class="details-container">
+    <div class="recipe-left">
+      <ul>
+        <li>Preparation time <%= @recipe.preparation_time %> hours</li>
+        <li>Cooking time <%= @recipe.cooking_time%> hours</li>
+        <li><%= @recipe.description%></li>
+      </ul>
+    </div>
+
+    <div class="recipe-right">
+      <% if (@recipe.user == current_user) %>
+        <div><%=@recipe.public? ? 'Public' : 'Private'%></div>
+        <%= button_to @recipe.public? ? 'Make private' : 'Make public', toggle_recipe_path(id: @recipe.id), method: :post, class: 'toggle_btn', remote: true, class: "btn-primary" %>
+      <% end %>
+    </div>
+  </div>
+</section>
+
+
+<div class="detail-links">
```

Comment:
- [ ]  The previous reviewer suggested that you add an ingredient and generate shopping list feature so that users could see the list of ingredients and create a shopping list. However, when I click on a recipe to check the ingredients and shopping list, I encounter an error that prevents me from doing so. Please fix the issue and make sure that your app has the requested features as previous reviewer suggested 
![image](https://github.com/Itswali/Recipe-App/assets/43877689/51328af2-7a2e-4a4f-9c21-d3340a974a0a)